test/Handler/SitesSpec.hs

```diff
@@ -108,3 +111,9 @@ spec = withApp $ do
 
 createSite_ :: UserId -> Site -> DB ()
 createSite_ userId = void . createSite userId
+
+selectLanguage :: Text -> RequestBuilder m ()
+selectLanguage lang = do
```

Comment:
Cool. If we find that helper later, we can use it. If not, this could be a good pull request to Yesod.Test down the line.